misc/dist/html/full-size.html

```diff
@@ -140,7 +140,7 @@
 		} else if (typeof err === 'string') {
 			setStatusNotice(err);
 		} else {
-			setStatusNotice('An unknown error occured');
+			setStatusNotice('An unknown error occurred');
```

Comment:
```suggestion
			setStatusNotice('An unknown error occurred.');
```
Possibly this. See other comment for reasoning